kong/config_service.go

```diff
@@ -37,29 +38,25 @@ func (c *ConfigService) ReloadDeclarativeRawConfig(
 	}
 	req, err := c.client.NewRequest("POST", "/config", sendConfigParams{CheckHash: checkHashI}, config)
 	if err != nil {
-		return fmt.Errorf("creating new HTTP request for /config: %w", err)
+		return []byte{}, fmt.Errorf("creating new HTTP request for /config: %w", err)
 	}
 
 	resp, err := c.client.DoRAW(ctx, req)
 	if err != nil {
-		return fmt.Errorf("failed posting new config to /config: %w", err)
+		return []byte{}, fmt.Errorf("failed posting new config to /config: %w", err)
 	}
 	defer resp.Body.Close()
 
+	var b []byte
 	if resp.StatusCode < 200 || resp.StatusCode >= 400 {
-		b, err := io.ReadAll(resp.Body)
+		b, err = io.ReadAll(resp.Body)
 		if err != nil {
-			return fmt.Errorf(
-				"failed posting new config to /config: got status code %d (and failed to read the response body): %w",
-				resp.StatusCode, err,
-			)
+			return nil,
```

Comment:
Now this is inconsistent with lines 46 and 41 🙃

Reply:
https://github.com/Kong/go-kong/pull/271/commits/6a88d74095232a710a6c08b7bc67aeab668e87e8